driver/bpf/fillers.h

```diff
@@ -2751,24 +2751,26 @@ FILLER(sys_unshare_e, true)
 
 FILLER(sys_generic, true)
 {
-	long *scap_id;
+	int libs_id;
```

Comment:
I would change it into something like `ppm_id` or `ppm_code` just to be more explicit. This is just my personal opinion feel free to ignore it, also `libs_id` sounds pretty well :)

Reply:
You're right, it makes sense. Really bad naming :) I could even leave `scap_id` there.

Reply:
Yeah, `scap_id` works for me :)

Reply:
@Andreagit97 done :)